nodejs/__test__/registry.test.ts

```diff
@@ -155,3 +145,79 @@ describe.each([arrow15, arrow16, arrow17, arrow18])("Registry", (arrow) => {
     expect(schema.metadata).toEqual(expectedMetadata);
   });
 });
+
+describe("Registry.setVar", () => {
+  const registry = getRegistry();
+
+  beforeEach(() => {
+    @register("mock-embedding")
+    // biome-ignore lint/correctness/noUnusedVariables :
```

Comment:
What's the unused variable here?

Reply:
The embedding function. The `@register()` decorator adds it to the registry, and we use it implicitly. Typescript doesn't seem to understand that.